src/scss/components/nav.scss

```diff
@@ -337,6 +337,32 @@
   }
 }
 
+.main {
+  .tags-list {
+    display: flex;
+    justify-content: center;
+  }
+
+  .tags-list__tag-item {
+    margin: 0.5rem;
+  }
+
+  .tags-list__tag-link {
+    color: white;
+    text-transform: uppercase;
+    padding: 0.2rem 0.5rem;
+    text-decoration: none;
+    font-size: 0.8rem;
```

Comment:
```suggestion
    font-size: .8rem;
```

Reply:
Seems fine, but are you getting this from a standard somewhere? We're not linting on it, and using 0.x for units is already in a number of places in watea.

Reply:
Some projects have it as a convention because it reduces the size of your CSS.

The 30 sec we collectively spent thinking about it probably already outweigh the benefit.